src/lib/misc.ts

```diff
@@ -22,8 +22,47 @@ import { browserHistory } from "ogsHistory";
 import * as preferences from "preferences";
 import swal from "sweetalert2";
 
+import { Goban } from "Goban";
```

Comment:
`from "goban"` instead of `from "Goban"` should fix the build issue :)

Reply:
Hah!   Will fix.   But ... why does it work locally?

Reply:
I fixed it already, I reckon it works for you because your filesystem isn't case sensitive ☺️

Reply:
Wah???   (runs away, hopes never to hear about this again).